mithril-client/src/aggregator_client.rs

```diff
@@ -22,7 +22,6 @@ use tokio::sync::RwLock;
 use mithril_common::entities::{ClientError, ServerError};
```

Comment:
These changes look unrelated. What are they?

Reply:
compiler complained about of unused dependency, it was easy to fix.

Reply:
When you make the PR upstream, make sure to put this information in the PR itself.
So it's clear it's not strictly part of the work to make the dependency selectable.